src/web/src/components/CourseList.vue

```diff
@@ -28,12 +28,33 @@
     <hr />
 
     <b-list-group id="scroll-box" class="mb-2 mb-sm-0 flex-grow-1" flush>
-     <b-list-group-item
+      <b-list-group-item
         v-for="course in filteredCourses"
         :key="course.id"
         :class="{'bg-light': course.selected}"
       >
-        <CourseListing :course="course" :actions="{add:true}" v-on="$listeners" />
+        <CourseListing :course="course" lazyLoadCollapse v-on="$listeners">
+          <template #toggleCollapseButton="{ course, toggleCollapse }">
+            <button
+              class="btn"
+              @click="toggleCollapse()"
+              :disabled="!course.corequisites && !course.prerequisites"
+            >
+              <font-awesome-icon :icon="faInfoCircle" />
+            </button>
+          </template>
+          <template #collapseContent="{ course: {corequisites, prerequisites, description} }">
```

Comment:
What does the hash sign do in this case?

Reply:
I believe its a shorthand for named slots so the alternative is `v-slot:name=`